client/packages/common/src/ui/components/modals/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -45,6 +47,13 @@ export const ConfirmationModal = ({
             <Typography variant="h6">{title}</Typography>
```

Comment:
Oh, just realised the `OK` button at the bottom of this file actually is hardcoded string `OK`! If you could update to translated value as part of these changes that would be great 🙏 

Could add new optional `buttonLabel` prop to `ConfirmationModal` to override the OK with `Continue` if you wanted to make Mariya real happy 😉

Reply:
Great, I have done this 😁